firefox-ios/Client/Frontend/Settings/Main/Support/SentFromFirefoxSetting.swift

```diff
@@ -0,0 +1,46 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Foundation
+import Shared
+
+class SentFromFirefoxSetting: BoolSetting {
```

Comment:
Do we plan to use this class for sub classing? Otherwise I'd suggest making it final.

Reply:
```suggestion
final class SentFromFirefoxSetting: BoolSetting {
```
Sure I can do that! 👍